woocommerce/woocommerce-android
# Introduce CardReaderHubFragment

Parent issue #4413 

This PR does more than one thing, but since this screen is extremely simple I thought it'd be ok (the number of changed lines is < 300). If you want to follow my train of thoughts, review this PR by commits.

This PR 
- introduces a new screen (if you can come up with a better name then "Hub" let me know ;) )

| Light | Dark |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/2261188/128044414-174c9c4f-f464-43b1-97dc-8c9cf49113bb.jpg" width="250px" /> | <img src="https://user-images.githubusercontent.com/2261188/128045082-22253f3b-f24b-45a6-bf90-f4d6d2c8dc94.jpg" width="250px" /> |

- Updates navigation from Onboarding flow to this new screen (previously the app was navigating the user to CardReaderDetailFragment)


To test:
- Make sure to use a site that is eligible for in-person payments and is ready to accept payments (wcpay is installed, stripe account setup etc)
1. Tap on app settings
2. Tap on "In-Person Payments"
3. Notice a loading screen is shown and you get redirected to the new Hub fragment
4. Note: "Order Card Reader" action is not implemented yet and the app crashes when you click on it
5. Tap on "Manage Card Reader" and notice you get redirected to CardReaderDetailFragment

Update release notes:
